wdl/wdl_options.proto

```diff
@@ -514,6 +516,9 @@ message WdlTraitOptions {
   optional uint32 reserved_command_tag_max = 11;
   optional uint32 reserved_event_tag_min = 12;
   optional uint32 reserved_event_tag_max = 13;
+
+  optional WdlPermission get_permission = 20;
+  optional WdlPermission update_permission = 20;
```

Comment:
Should be different field numbers

Reply:
Also, these fields seem more like commands rather than state variables.

Reply:
Fixed the field tag.